openbas-model/src/main/java/io/openbas/database/repository/UserRepositoryImpl.java

```diff
@@ -0,0 +1,48 @@
+package io.openbas.database.repository;
+
+import io.openbas.database.raw.BasicOrganization;
+import io.openbas.database.raw.BasicUser;
+import jakarta.persistence.EntityManager;
+import jakarta.persistence.PersistenceContext;
+import jakarta.persistence.Tuple;
+import java.sql.Timestamp;
+import java.util.Arrays;
+import java.util.List;
+
+public class UserRepositoryImpl implements UserRepositoryCustom {
+
+  @PersistenceContext
+  private EntityManager em;
+
+
+  @Override
+  public List<BasicUser> rawAll() {
```

Comment:
Why didn't you follow the proposed solution by Julien ?

Reply:
In groupPopover, we need user.organisation.xx (other level), so I should build a custom mapper for this object
![image](https://github.com/OpenBAS-Platform/openbas/assets/4162296/d392ef86-75b3-4dc5-a129-942a82637752)